examples/catalyst_contrastive/common.py

```diff
@@ -0,0 +1,11 @@
+# flake8: noqa
+from catalyst import dl
+
+
+class ContrastiveRunner(dl.SupervisedRunner):
+    def handle_batch(self, batch):
+        # model train/valid step
+        # unpack the batch
+        emb1 = self.model(batch["aug1"])
+        emb2 = self.model(batch["aug2"])
+        self.batch = {"proj1": emb1, "proj2": emb2}
```

Comment:
```suggestion
        sample_aug1, sample_aug2, tatget = batch
        projection1 = self.model(sample_aug1)
        projection2 = self.model(sample_aug2)
        self.batch = {"projection1": projection1, "projection2": projection2}
```
for tuple-based dataset